Moise1/SendIt
# Ch -create order #162149652

**What Does This PR do?**
A client should be able to create parcel orders

**Description of tasks to be completed**

_A client should be able to create parcel orders._

**How should this be tested manually?**


After cloning the repo, cd into it and run "npm run dev-start" from the terminal

Click on "Order Now" from the nav bar, be directed to "Order Now" .
Click on the "order now" button to create a parcel order . Then in address bar, change your url to lik like this:  **https://www.sendit.com/api/v1/parcels** . Then, you should be to able to view the parcel order added to the parcels list .

**What the relevant Pivotal Tracker stories?**
#162149652
